package.json

```diff
@@ -11,13 +11,16 @@
     "teachcode": "./bin/index.js"
   },
   "scripts": {
-    "lint": "eslint src bin test",
-    "lint:fix": "eslint --fix src bin test",
+    "lint": "eslint src bin __e2e__",
+    "lint:fix": "eslint --fix src bin __e2e__",
     "pretest": "npm run lint",
-    "test": "ava test/index.js",
+    "test": "ava --serial",
```

Comment:
`ava` tests run serially as tests from different files may alter the same `config.json` file from the filesystem.